pinot-core/src/main/java/com/linkedin/pinot/core/data/extractors/PlainFieldExtractor.java

```diff
@@ -113,6 +129,15 @@ public GenericRow transform(GenericRow row) {
       boolean hasNull = false;
       boolean hasConversion = false;
       for (String column : _schema.getColumnNames()) {
+        // Adding outgoing time column to fieldMap;
+        if (column.equals(_schema.getTimeColumnName())) {
+          Object incomingTimeValue = row.getValue(_incomingTimeColumnName);
+          if (incomingTimeValue == null) {
+            incomingTimeValue = _schema.getTimeFieldSpec().getDefaultNullValue();
+          }
+          fieldMap.put(_outGoingTimeColumnName, _timeConverter.convert(incomingTimeValue));
+          continue;
```

Comment:
remove this, the rest of the code should handle this correct, it has lot of things like setting default value, counting error rows etc